src/components/Header.styles.tsx

```diff
@@ -23,23 +18,20 @@ export const Button = styled.button`
     cursor-pointer
     font-bold
     rounded
-    px-4
+    px-3
     py-2
-    mr-2
-  `}
+    mr-1
+    border-0
+    text-white
+    text-xs
+    font-mono
+    bg-[#1d1e1f]
```

Comment:
I think it's possible to define your own colors in `tailwind.config.js`
https://tailwindcss.com/docs/customizing-colors

Maybe something like?
```js
module.exports = {
  theme: {
    colors: {
      'bnomial-black': '#1d1e1f',
      'bnomial-gray': '#565758',
      ...
    },
  },
};
```

Maybe it's also good to ask Denny for an official (maybe even named) palette

Reply:
Yes, I know and I've done this with the fonts, but I didn't do it yet because I wanted to think a bit more about organizing the color pallet.

Reply:
Ok well then this can stay as an open issue I guess for now ;)